sway-lsp/Cargo.toml

```diff
@@ -65,8 +68,8 @@ pretty_assertions = "1.4.0"
 rand = "0.8"
 regex = "^1.10.2"
 sway-lsp-test-utils = { path = "tests/utils" }
-tikv-jemallocator = "0.5"
-tower = { version = "0.4.12", default-features = false, features = ["util"] }
+tikv-jemallocator = "0.6.0"
```

Comment:
I personally prefer using "x.y" instead of "x.y.z" for dependency versions, as it automatically incorporates bug fixes and security updates within the x.y.* range. What are your thoughts about adopting this approach?